setup.cfg

```diff
@@ -0,0 +1,10 @@
+[isort]
+known_first_party = deoplete_jedi
+default_section = THIRDPARTY
+lines = 79
```

Comment:
Now using `99` for lines on flake8 config. I think confirm to it is better.
https://github.com/zchee/deoplete-jedi/blob/master/.flake8#L17